docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '2'
+services:
+  mariadb:
+    image: docker.io/bitnami/mariadb:10.5
+    environment:
+      - ALLOW_EMPTY_PASSWORD=yes
+      - MARIADB_USER=bn_moodle
+      - MARIADB_DATABASE=bitnami_moodle
+      - MARIADB_CHARACTER_SET=utf8mb4
+      - MARIADB_COLLATE=utf8mb4_unicode_ci
+    volumes:
+      - 'mariadb_data:/bitnami/mariadb'
+
+  moodle:
+    image: docker.io/bitnami/moodle:3.11.2
+    ports:
+      - '8080:8080'
+      - '443:8443'
+    environment:
+      - MOODLE_DATABASE_HOST=mariadb
+      - MOODLE_DATABASE_PORT_NUMBER=3306
+      - MOODLE_DATABASE_USER=bn_moodle
+      - MOODLE_DATABASE_NAME=bitnami_moodle
+      - ALLOW_EMPTY_PASSWORD=yes
+      - ACCREDIBLE_DEV_API_ENDPOINT=http://host.docker.internal:3000/v1/
+    volumes:
+      - 'moodle_data:/bitnami/moodle'
+      - 'moodledata_data:/bitnami/moodledata'
+      # Install Accredible plugin by default
+      - '.:/bitnami/moodle/mod/accredible'
```

Comment:
As discussed, there is an issue with adding the accredible plugin on a new container. The moodle service fails to start in this scenario.